README.md

```diff
@@ -1 +1,34 @@
-# secrethub-examples
\ No newline at end of file
+<p align="center">
+  <img src="https://secrethub.io/img/secrethub-logo.svg" alt="SecretHub" width="380px"/>
+</p>
+<h1 align="center">
+  <i>Examples</i>
+</h1>
+
+[![Version](https://img.shields.io/github/release/secrethub/demo-app.svg)](https://github.com/secrethub/demo-app/releases/latest)
+[![Discord](https://img.shields.io/badge/chat-on%20discord-7289da.svg?logo=discord)](https://discord.gg/NWmxVeb)
+
+This repository contains example integrations with SecretHub. The README.md for these examples can be found in every directory. For more information, take a look at our [integrations page](https://secrethub.io/docs/#integrations).
+
+We're adding more examples over the time, so keep a look out for new examples. If you have a great example of a SecretHub integration yourself or a way to improve an example, feel free (and very welcome) to make a pull request.
+
+## Table of contents
+
```

Comment:
Should we add a note that the ones without a link are coming soon but not yet available?